livekit-plugins/livekit-plugins-cartesia/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "livekit-plugins/livekit-plugins-cartesia",
```

Comment:
this is because we don't know which packages are and which aren't under livekit-plugins unless we do this. it's easier to strip it in the CI than to figure out whether we need to add it

Reply:
it's used programmatically to know where to cd in the CI

Reply:
maybe we can add a comment either in the publish-package.yml (or in the README) about what naming format we expect/need for this logic to work. Just as a reference once we add more lateron.

Reply:
maybe in the README for livekit-plugins-minimal, since that might be copied over as boilerplate for new packages?

Reply:
sounds good, the root README sounds also appropriate

Reply:
having trouble fitting this anywhere in the root README, it feels too end-user focused and this is internal dev doc
i think the plugin part is good, i don't assume we'll add anything to anywhere but the plugin directory in this repository